pipeline-maven/src/test/java/org/jenkinsci/plugins/pipeline/maven/AbstractIntegrationTest.java

```diff
@@ -44,6 +46,14 @@ public abstract class AbstractIntegrationTest {
 
     String mavenInstallationName;
 
+    public static GenericContainer<?> createContainer(String target) {
+        return new GenericContainer<>(new ImageFromDockerfile(
+                                "localhost/pipeline-maven/" + target, Boolean.parseBoolean(System.getenv("CI")))
+                        .withFileFromClasspath(".", "/org/jenkinsci/plugins/pipeline/maven/docker")
```

Comment:
I do not get it ... why creating multiple image, with different names, if they all have the content ?

why merging all the different contents into only one image ?

Reply:
Sorry, missing coffee, not fully awake ...

I missed the multi staged :man_facepalming: